appium/lib/screen-object/accessors/element.rb

```diff
@@ -174,14 +183,15 @@ def swipe_element_right
         swipe_screen_element(:right, 2000)
       end
 
-      def scroll_for_dynamic_element_click (expected_text)
+      def scroll_for_dynamic_element_tap (expected_text)
         if dynamic_xpath(expected_text).displayed?
           click
         else
           scroll
           click
         end
       end
+      alias_method :scroll_for_dynamic_element_click, :scroll_for_dynamic_element_tap
```

Comment:
You will have to update these as well after updating name to `scroll_dynamic_text_to_tap`